docker/archimedes-base/Dockerfile

```diff
@@ -1,3 +1,6 @@
+FROM magby
```

Comment:
Multi-stage builds are essentially when one docker file refers to some other docker image to borrow resources from it.

Typically, `FROM` statements are used to describe dependencies between docker images inside of dockerfiles creating new docker images.  We're leveraging this syntax here to say "hey, go find a project directory called magby and have its image built` (this is done by code in `utils.mk`), then I want to copy some files from that updated image" (you'll see below).